.github/workflows/pr_validation.yaml

```diff
@@ -64,11 +64,19 @@ jobs:
           echo
           echo
           echo "Difference of com.canonical.certification::sru-server (diff this_branch main)"
+          # the following line sorts the array of jobs/templates obtained from the pre/post checkout
+          # this is done because we want compare the list/content of jobs but not necessarely the order
+          # that for most jobs is not deterministic either way. The array keys are sorted as well
+          # as what counts is the content of each key (id, environ, command) and not the order
+          # the exporter prints them out
```

Comment:
```suggestion
          # The following line sorts the array of jobs/templates obtained from the pre/post checkout.
          # This is done because we want compare the list/content of jobs but not necessarily the order
          # that for most jobs is not deterministic either way. The array keys are sorted as well
          # as what counts is the content of each key (id, environ, command) and not the order
          # the exporter prints them out.
```